content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -55,8 +55,8 @@ to use this feature with Kubernetes stateless pods:
 * CRI-O: version 1.25 (and later) supports user namespaces for containers.
 
 Please note that containerd v1.7 supports user namespaces for containers,
-compatible with Kubernetes {{< skew currentPatchVersion >}}. It should not be used
-with Kubernetes 1.27 (and later).
+compatible with Kubernetes 1.25 and 1.26. It should not be used with Kubernetes
+1.27 (and later).
```

Comment:
Try either this:
```suggestion
<!-- this should be checked at each Kubernetes release and each containerd to verify that it is
correct -->
{{< note >}}
Kubernetes {{< skew currentVersion >}} does not support user namespaces when used with
containerd. However, some previous versions may offer compatibility when used with specific
containerd versions. If you are using a version of Kubernetes earlier than {{< skew currentVersion >}},
check the documentation for that version of Kubernetes.
{{< /note >}}
```
or, drop the list of compatible container runtimes (for example, if we don't have capacity to maintain it).

Reply:
As I explained, this is not needed at all. See: https://github.com/kubernetes/website/pull/41906#issuecomment-1625184829

Reply:
As a reviewer for English, I've cancelled the mark that this is a resolved comment.

Reply:
@sftim can you make a suggestion that doesn't say that we need to revisit this on every release, then?

Reply:
An alternative is to say something like:
```markdown
containerd v1.7 is not compatible with the userns support in Kubernetes v{{< skew currentVersion >}}.
Kubernetes v1.25 and v1.26 used an earlier implementation that **is** compatible with containerd v1.7,
in terms of userns support.
If you are using a version of Kubernetes other than {{< skew currentVersion >}},
check the documentation for that version of Kubernetes for the most relevant information.
If there is a newer release of containerd than v1.7 available for use, also check the containerd
documentation for compatibility information.
```

Reply:
@sftim `skew currentVersion` expands to the latest patch release, which is not ideal IMHO: is kubernetes 1.27.3 not compatible but maybe 1.27.2 was?

I don't see an advantaje to use the variables instead of `1.27 (and later)`. Is there any?

Reply:
> @sftim `skew currentVersion` expands to the latest patch release, which is not ideal IMHO: is kubernetes 1.27.3 not compatible but maybe 1.27.2 was?

Have you tested this?

Reply:
You might be thinking of the existing shortcode, `{{< skew currentPatchVersion >}}`.

Reply:
Oh, right, I was thinking of that. Updates to this, then!